packages/orn-locator/src/ornPatterns.ts

```diff
@@ -20,26 +20,25 @@ export const patterns = {
   }),
   book: makePattern({
     name: 'Books',
-    pattern: 'https\\://openstax.org/orn/book/:id',
-    resolve: ({id}: {id: string}) => import('./resolvers/books').then(mod => mod.bookDetail(id)),
+    pattern: 'https\\://openstax.org/orn/{archive/:archivePath/}?book/:id{@:version}?',
```

Comment:
I think this works as long as we force the book content and archive versions to be specified together, otherwise the `:` is ambiguous with the other `:` in the pattern

Reply:
Changed to this format, but you can only specify the archive version if you also specify the book version.